### PR TITLE
Add Gravity

### PR DESCRIPTION
**Pull request by antooni**

### CODE REVIEW

**Review by luke7211 — CHANGES_REQUESTED** [review]
Check current token list and add major tokens missing (USDT, WBTC, maybe PSTAKE)

**Comment by sz-piotr** [comment]
I have update the risk view descriptions to be in line with #794. Please check if the changes are correct before proceeding

**Comment by antooni** [comment]
I decided not to include PSTAKE because it is currently 635th by mc on coingecko, so it fells below our top-300 coingecko criteria

File: packages/config/src/bridges/gravity.ts [hunk 1]
```diff
@@ -9,20 +10,94 @@ export const gravity: Bridge = {
...
+      },
+    ],
+    risks: [],
+    isIncomplete: true,
+  },
+  permissions: [
+    {
+      name: 'Cosmos Validators',
+      description:
+        'Control Gravity contract on Ethereum, funds cannot be transfer without the signature of at least 2/3 of the validators set.',
```
Comment (sz-piotr) [inline packages/config/src/bridges/gravity.ts:99]: How does Ethereum know who the 2/3 are

**Review by luke7211 — APPROVED** [review]

**Pull request merged**